azurerm/internal/services/network/tests/web_application_firewall_policy_data_source_test.go

```diff
@@ -0,0 +1,127 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+)
+
+func TestAccDataSourceAzureRMWebApplicationFirewallPolicy_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "data.azurerm_web_application_firewall_policy", "test")
+	resourceGroupName := fmt.Sprintf("acctestRG-%d", data.RandomInteger)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { acceptance.PreCheck(t) },
+		Providers: acceptance.SupportedProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceAzureRMWebApplicationFirewallPolicyBasic(data),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(data.ResourceName, "name", "example-wafpolicy"),
+					resource.TestCheckResourceAttr(data.ResourceName, "resource_group_name", resourceGroupName),
+					resource.TestCheckResourceAttr(data.ResourceName, "tags.%", "1"),
+					resource.TestCheckResourceAttr(data.ResourceName, "tags.env", "test"),
+				),
+			},
+		},
+	})
+}
+
+func testAccDataSourceAzureRMWebApplicationFirewallPolicyBasic(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_web_application_firewall_policy" "example" {
+  name                = "example-wafpolicy"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
```

Comment:
this is "test" above - could we make this:

```suggestion
  location            = azurerm_resource_group.test.location
```